Bottom: 445851b5543e1a85d2f2472e7b6216876661a75c
Top:    d15c7bb2ecee7da0d32377a790ffe463608870b9
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2012-03-31 09:55:27 -0400

Refresh of device-trees

---

diff --git a/dt_config b/dt_config
index 946862d..8001b7b 100644
--- a/dt_config
+++ b/dt_config
@@ -730,6 +730,9 @@ CONFIG_BLK_DEV_XIP=y
 # Altera FPGA firmware download module
 #
 # CONFIG_ALTERA_STAPL is not set
+CONFIG_NEW_LPC313x_IO=y
+CONFIG_LPC313x_ADC=y
+CONFIG_LPC313x_PWM=y
 
 #
 # SCSI device support
@@ -1254,7 +1257,7 @@ CONFIG_SND_VERBOSE_PROCFS=y
 CONFIG_SND_SOC=y
 CONFIG_SND_LPC313X_SOC=y
 CONFIG_SND_LPC313X_SOC_I2S=y
-CONFIG_SND_LPC313X_SOC_I2S_UDA1380=y
+CONFIG_SND_EA3131_SOC_I2S_UDA1380=y
 CONFIG_SND_USE_EA3131=y
 CONFIG_SND_I2C1_CHANNEL_UDA1380=y
 CONFIG_SND_I2S_RX0_MASTER=y
